python/lsst/analysis/ap/apdbUtils.py

```diff
@@ -0,0 +1,99 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Utility functions for connecting to, and manipulating the output from, APDB.
+"""
+
+import sqlite3
+import psycopg2
+
+
+def connectToApdb(dbName, dbType='sqlite', schema=None):
```

Comment:
I lifted this code from ap_pipe-notebooks, as I said in my other comment. I'm not sure I want to go and design a new APDB API on this ticket, since I don't really know much about use cases or what the interface really should look like.

Can I file a ticket about making a better API and put a TODO on this?

Reply:
As @kfindeisen you can use the existing Apdb interface for to access the data, it is however designed with spatial loading in mind using HTM indices. You can see how I use the interface in `ap_association` in the  [loadDiaCatalog](https://github.com/lsst/ap_association/blob/master/python/lsst/ap/association/loadDiaCatalogs.py) tasks. Mostly, Apdb uses the Pandas read/write functions for loading and storing data in SQL as of a while now.

Reply:
Ok, then maybe it's better to leave it as-is, at least until [DM-31458](https://jira.lsstcorp.org/browse/DM-31458) (which makes various API changes) lands.